deploy/ec2/setup_app

```diff
@@ -8,7 +8,7 @@ fi
 
 export $(grep -v '^#' .env | xargs)
 
-if psql -d postgresql://$PG_USER:$PG_PASS@$PG_HOST -c 'select now()' > /dev/null 2>&1
+if psql -d postgresql://$PG_USER:$PG_PASS@$PG_HOST/$PG_DB -c 'select now()' > /dev/null 2>&1
```

Comment:
Hey @MehdiElAouni, could you change the `$PG_DB` value to `postgres` rather? It is default db which can be used to check for connection. In the current change, it can fail when the particular db isn't created yet.